src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Collaborate with your teammates
+tags:
+  - New Relic 
+  - Use New Relic
+  - Get started
+metaDescription: "Our new collaboration features help you use New Relic to solve problems together"
+---
+
+import close_and_reopen_discussions from 'images/close-and-reopen-discussions.png'
+
+import collaboration_discussion_board_new from 'images/collaboration-discussion-board-new.png'
+
+import green_dot from 'images/green_dot.png'
+
+<Callout variant="important">
+The functionality described below is currently in preview. If you don't yet have access but would like to test the commenting and discussion functionality [please sign up here](https://one.newrelic.com/discussions).
+</Callout>
+
+We're making it easier to collaborate with your teammates by building discussion and commenting tools into New Relic. You can now add comments on any page. Our integrations with Slack, MS Teams (coming soon), and email let you share links and screenshots, and communicate with your teammates directly from the New Relic UI without switching applications. Our collaboration tools allow you to identify problems and find solutions in one place. Moving the discussion closer to your data streamlines your workflows, improves efficiency, and reduces MTTR.
```

Comment:
Not sure there needs to be a comma between "links and screenshots" and "and communicate"